src/classes/Api.js

```diff
@@ -61,14 +70,43 @@ class API {
 
   async resetDatabase() {
     let { status } = (await axiosInstance.get("list/reset-database")).data;
```

Comment:
```suggestion
    const status = (await axiosInstance.get("list/reset-database")).data;
```
Is it a requirement to use `{ var }` in this case?

Reply:
Nah, it's fine in both cases, just shorter syntax.

Reply:
`let { status } = (await axiosInstance.get("list/reset-database")).data;`
is actually equal to 
`const status = (await axiosInstance.get("list/reset-database")).data.status;`

Reply:
Alright. I'll use you suggestion.